src/__test__/utils/sendWork.test.js

```diff
@@ -29,7 +29,20 @@ jest.mock('../../utils/socketConnection', () => {
   };
 });
 
-describe('sendWork unit tests', () => {
+jest.mock('aws-amplify', () => ({
+  configure: jest.fn().mockImplementation(() => ({
+    Storage: {
+      AWSS3: {
+        bucket: 'biomage-originals-test',
```

Comment:
how about we put this bucket name in config? And in here, we get the config value for it instead of hardcoding it?